_opensearch/install/important-settings.md

```diff
@@ -38,3 +38,17 @@ The [sample docker-compose.yml]({{site.url}}{{site.baseurl}}/opensearch/install/
 - `port 9600`
 
   Allows you to access Performance Analyzer on port 9600.
+
+### Network requirements
+
+  The following firewall ports need to be open for OpenSearch components.
+
+Port number | OpenSearch component
+:--- | :--- 
+443 | OpenSearch Dashboards
```

Comment:
Just a thought. . if both 443 and 5601 are required to use the Dashboards in all cases, maybe indicate that both are required. If not, perhaps indicate when they would need both. (e.g. optionally both are required if <use case or condition> exists).

Reply:
Remove "firewall".

Reply:
Added distinction between 443 and 5601